test/unit/resources/azure_aks_cluster_test.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'helper'
```

Comment:
Can you double-check that this file is valid in your local IDE. I'm not seeing any syntax highlighting in the GitHub UI.

Reply:
Yes, it is a valid file: https://github.com/inspec/inspec-azure/blob/master/test/unit/resources/helper.rb